### DISCUSSION TITLE
fluentui/react-monaco-editor still recommended not to use?

### DISCUSSION THREAD

**Discussion by staubt** [microsoft/fluentui#17235] [Q&A] [answered]
In the 4 month old readme file is still a recommendation to not use the lib outside of the fluent react repo. 
**WARNING**: The editor component's API is still **highly unstable**, so it **should not** be used outside the Fluent UI React repo yet.
Is that still valid?

Best regards

**Comment by JustSlone** [comment, marked as answer]
Yep, while it's not exactly unstable (given we're not actively developing it). We aren't able to support it for other parties to use it, and it will be removed at some point when we move to a better website. 

If you like it and want to use it, feel free to fork it though!
